homeassistant/components/yweather/sensor.py

```diff
@@ -0,0 +1,200 @@
+"""Support for the Yahoo! Weather service."""
```

Comment:
Should this entire file be part of this PR?

Reply:
No. Please remove this file.